src/Visualization/Display3d.cpp

```diff
@@ -46,28 +46,18 @@ void Display3d::Init(long window_handle,
   printf("Aspect_DisplayConnection created.\n");
   if (GetGraphicDriver().IsNull())
   {
-  GetGraphicDriver() = new OpenGl_GraphicDriver (aDisplayConnection);
+    GetGraphicDriver() = new OpenGl_GraphicDriver (Handle(Aspect_DisplayConnection)());
```

Comment:
@jf--- why do you change these lines in Display3d. I recently refactored the Display3d class I can' understand why you revert those changes

-1